tools/integration_tests/concurrent_operations/setup_test.go

```diff
@@ -62,7 +62,7 @@ func TestMain(m *testing.M) {
 	setup.SetUpTestDirForTestBucketFlag()
 
 	flagsSet := [][]string{
-		{"--kernel-list-cache-ttl-secs=-1"},
+		{"--kernel-list-cache-ttl-secs=-1", "--kernel-list-cache-ttl-secs=0"},
 	}
```

Comment:
This is only for concurrent tests right. Not for other tests right, because there are tests which may fail due to list caching.

Reply:
Yes only for concurrent operations.